core/src/components/content/content.tsx

```diff
@@ -188,7 +190,9 @@ export class Content implements ComponentInterface {
     if (dirty) {
       this.cTop = top;
       this.cBottom = bottom;
-      forceUpdate(this);
+      if (Build.isBrowser) {
```

Comment:
`readDimensions` is only called by `resize` which already has the `Build.isBrowser` check. Do we need another `Build.isBrowser` check here?